test/test_torch.py

```diff
@@ -313,6 +313,38 @@ def test_dim_reduction_less_than_64(self):
         with self.assertRaisesRegex(RuntimeError, "PyTorch doesn't support reduction operations for dim>=64"):
             torch.sum(x, -1)
 
+    def _test_logaddexp(self, base2=False):
```

Comment:
@albanD What's our plan for testing the derivatives of new functions like this?

Reply:
I am also curious.